recipes/full_finetune_distributed.py

```diff
@@ -121,6 +121,12 @@ def __init__(self, cfg: DictConfig) -> None:
         self._log_every_n_steps = cfg.get("log_every_n_steps", 1)
         self._log_peak_memory_stats = cfg.get("log_peak_memory_stats", False)
 
+        if self._log_peak_memory_stats and self._device.type == "cuda":
+            log.info(
+                "log_peak_memory_stats was se to True, however, training does not use cuda. Setting log_peak_memory_stats=False."
```

Comment:
i think that there are typos. Thats my fault, i guess you just copied/pasted what i wrote earlier.

Reply:
Yes))) I didn't double check because it pretty minor

Reply:
Fixed typos

Reply:
@krammnic sorry, just realized that the condition is wrong

Reply:
Good catch! Yeah need to be fixed